components/Branding/LogoSection/LogoSection.js

```diff
@@ -12,158 +12,124 @@ function LogoSection() {
       theme="gray"
       hasTitleUnderline
       columns={[
-        <ul className={styles.logoSizeList}>
+        <dl className={styles.logoSizeList}>
```

Comment:
dl and dt is incorrect semantically. It's not a definition list.

Reply:
From the spec:

```
Use ol as a container element for an ordered list of items.
Use ul as a container element for an unordered list of items.
Use li elements to identify items in ordered (ol) and unordered (ul) lists.
Use dl as a container element for a description list (a.k.a. definition list) that contains contiguous groupings of terms and their associated descriptions or definitions.
```

If defining how to use logos isn't a definition list why would an unordered list be semantically better?

Reply:
A definition list would be like a dictionary or a glossary. This is not that.

Reply:
Is this going to prevent being merged because we can go circles arguing about semantics